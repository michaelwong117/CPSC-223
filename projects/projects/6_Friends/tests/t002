#!/bin/bash
# Querying the ends of a graph consisting of two unconnected components

trap "/usr/bin/killall -q -u $USER ./Friends 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Friends ]; then
  echo './Friends is missing or not executable'
  echo './Friends is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./Friends /c/cs223/hw6/Tests/graph_alpha_subgraphs_2.in distance < /c/cs223/hw6/Tests/t002.in
