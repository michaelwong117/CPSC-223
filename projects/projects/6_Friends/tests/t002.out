A, E are not connected
